plugins/org.eclipse.n4js.cli/src/org/eclipse/n4js/cli/N4jscMain.java

```diff
@@ -34,11 +34,14 @@ public static void main(String[] args) {
 
 		final N4jscOptions options = getOptions(args);
 
-		if (!options.isVerbose()) {
+		if (options.isVerbose()) {
+			Logger.getRootLogger().setLevel(Level.DEBUG);
```

Comment:
Somehow, this logging state got lost in the past. I also checked how to see logging in tests, but this was not easy since the test infrastructure does not listen to any Logger.LOG output.